CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contributing to FFMD
+
+We are always happy about contributions towards our project. Before you set out on your own, please consider the following:
+
+* If you are from the Magdeburg/Germany area, visit our [meetings](http://www.netz39.de/events/) "Freifunk Bastelabend" and "Freifunk Abstimmungstreffen" to talk about the roadmap and current development efforts.
+* Contact us via [E-Mail or Jabber](http://www.netz39.de/verein/kommunikation/)
+* If your contribution is not specific to the Freifunk Magdeburg community, please have a look at the respective project, e.g. [gluon](https://github.com/freifunk-gluon/gluon).
+
+To contribute, please follow these steps:
+
+1. Create a branch based off `master` with a meaningful name.
+2. When your contribution is ready, create a pull request. If you would like comments on the way, prepend "WIP: " to your PR title.
+3. If you contribution should appear in the Changelog, please include a distinct commit adding the respective Changelog entry.
+4. We will review the code and there will be comments. Please be prepared for a dialogue. We welcome all contributions, but we also care for quality and making as few errors as possible, so we may request changes or clarifications. Bear with us so that we can come to a nice result.
```

Comment:
Suggestion: ".. and we also care"...

Reply:
"We welcome all contributions and we also care for quality and making as few errors as possible" ?

Reply:
@christf Vorschlag: https://github.com/FreifunkMD/site-ffmd/pull/132/commits/b34cd8459a855aecf8b268a56d4d15531c565eab